File: python/Problem1.py
'''
Created on Aug 19, 2011

@author: troot
'''


'''Find the sum of all the multiples of 3 or 5 below 1000.'''

final_sum_total = 0
i = 0

for i in range(1000):
	if (i%3) == 0 or (i%5) == 0:
		final_sum_total = final_sum_total + i
	
	i = i + 1

print "The answer is: %s" % final_sum_total

File: node/problem-1.js
/*
@author: troot
*/


// Find the sum of all the multiples of 3 or 5 below 1000.
const nArrLength = 1000;
const summed = new Array(nArrLength)
  .fill(0)
  .map((val, i, v) => i)
  .filter((val, i) => {
    return (((i % 3) === 0) || ((i % 5) === 0))
  })
  .reduce((acc, val) => acc + val, 0);

console.log('Filtered array value: ', summed);

